src/snowflake/cli/_plugins/nativeapp/sf_sql_facade.py

```diff
@@ -592,7 +592,7 @@ def upgrade_application(
 
         @param name: Name of the application object
         @param install_method: Method of installing the application
-        @param stage_fqn: FQN of the stage housing the application artifacts
+        @param stage_path_to_artifacts: Path to directory in stage housing the application artifacts
```

Comment:
nit: it's called version directory in the public docs

Reply:
ah I debated using that but I didn't. updated.